homeassistant/components/hassio/http.py

```diff
@@ -75,48 +72,28 @@ async def _handle(
 
     async def _command_proxy(
         self, path: str, request: web.Request
-    ) -> web.Response | web.StreamResponse:
+    ) -> web.StreamResponse:
         """Return a client request with proxy origin for Hass.io supervisor.
 
         This method is a coroutine.
         """
-        read_timeout = _get_timeout(path)
-        client_timeout = 10
-        data = None
         headers = _init_header(request)
         if path in ("snapshots/new/upload", "backups/new/upload"):
             # We need to reuse the full content type that includes the boundary
             headers[
                 "Content-Type"
             ] = request._stored_content_type  # pylint: disable=protected-access
```

Comment:
Is this override still needed with this approach?

Reply:
This PR didn't change anything about how the request looks to the supervisor (just how it sends over the bytes), so if it was needed before its still needed.
If I'm understanding correctly *why* this is needed, to include the file form data boundary parameter, I think it could come from request.raw_headers and doesn't need be specific to the /*/new/upload methods.